languages/cpp/src/main/java/de/jplag/cpp/CPPLanguage.java

```diff
@@ -50,4 +50,9 @@ public boolean supportsNormalization() {
     public List<Token> parse(Set<File> files, boolean normalize) throws ParsingException {
         return new CPPParserAdapter().parse(files);
     }
+
+    @Override
+    public boolean hasPriority() {
+        return true;
```

Comment:
```suggestion
        return true; // Priority over the C language module.
```
Let us document the interdependencies.